### ISSUE TITLE
Support for `FORCE_COLOR`

### ISSUE THREAD

**Issue by cardil** [fatih/color#155]
It would be a good idea to apart from supporting [`NO_COLOR`](https://nocolor) it would be good to support easy forcing of color output.

The `FORCE_COLOR` environmental variable is less spread across industry, but still is used in a couple of libraries, i.e. https://github.com/chalk/chalk#supportscolor

**Comment by pellared** [comment]
I strongly disagree that this feature should be implemented as there are no common standards regarding `FORCE_COLOR`.

If in future the `FORCE_COLOR` would be a standardized, then a change in its value handling would be a breaking change from the behavior perspective.

**Comment by juanzolotoochin** [comment]
For anyone else who finds this issue, I'm using this workaround:
```
import "github.com/fatih/color"

func ForceColor() {
	color.NoColor = false
}
```

Then I just call this at the top of `main()`

**Comment by pellared** [comment]
`FORCE_COLOR` becomes a standard: https://force-color.org/

**Comment by pellared** [comment]
I think it may be worth to revive https://github.com/fatih/color/pull/156

**Comment by cloudyparts** [comment]
`FORCE_COLOR` option is needed for CI jobs.  Gitlab CI supports colorized output in job logs.  CI Jobs TERM is set to "dumb" which mutes colors in this package.

**Comment by cardil** [comment]
@fatih Would you reconsider [your opinion](https://github.com/fatih/color/pull/156#issuecomment-1387454432), as https://force-color.org becoming a standard?
